modules/src/movie.py

```diff
@@ -33,9 +36,16 @@ def process(input, entities):
             })
             data = r.json()
 
+        # fetch the rating from IMDB
+        imdb_id = data['imdb_id']
+        imdb_search = ia.search_movie(imdb_id)
```

Comment:
The `search_movie` method is useful when we are querying the movie using keywords. Here, we already have the IMDb ID. We can directly use the `get_movie` method here.